src/dataviews/dataview-model-base.js

```diff
@@ -71,21 +78,16 @@ module.exports = Model.extend({
   initialize: function (attrs, opts) {
     attrs = attrs || {};
     opts = opts || {};
+    util.checkRequiredOpts(opts, REQUIRED_OPTS, 'DataviewModelBase');
+    util.setAsPrivateProperties(opts, this);
```

Comment:
I still don't buy it... IMO, this makes it hard to know what are the "private" properties of this class unless you actually go look for them in the actual code... I'd rather be explicit and make sure we're only declaring the variables that are indeed being used...

Reply:
OK. If you feel uncomfortable we could leave it only with `checkRequiredOpts` and then assign them manually.

But I must say that you concern is only caused by the novelty of seeing it. In practice, we haven't suffered any issues in Builder with this.

Reply:
Reverted.